updates/version.yaml

```diff
@@ -58,4 +58,6 @@
 4.6.1:
     - "Fix overwritten sitemap and the canonical tag is always added with the url of the current page"
 4.7.0:
-    - "Add Schema component"
\ No newline at end of file
+    - "Add Schema component"
+4.7.1:
+    - "Fix Favicon and Schema syntax error"
```

Comment:
change version comment to something more relevant to what you did in this task, since you've created webmanifest and that was main clue of task write something about it, syntax error is the least important here.